### PR TITLE
BackOfficeAsset custom fields

### PR DESCRIPTION
**Pull request by mkurek**
Added custom fields to BackOfficeAsset admin.

### CODE REVIEW

File: src/ralph/back_office/admin.py [hunk 1]
```diff
@@ -76,6 +77,7 @@ class BackOfficeAssetAdmin(
     BulkEditChangeListMixin,
     TransitionAdminMixin,
     AssetInvoiceReportMixin,
+    CustomFieldValueAdminMixin,
```
Comment (sputnikci) [inline src/ralph/back_office/admin.py]: \[Pylint\] INFO: Wrong hanging indentation before block \(add 4 spaces\)\.
    CustomFieldValueAdminMixin,
    ^   | \[bad\-continuation\]

Comment (sputnikci) [inline src/ralph/back_office/admin.py]: \[Pylint\] INFO: Wrong hanging indentation before block \(add 4 spaces\)\.
    CustomFieldValueAdminMixin,
    ^   | \[bad\-continuation\]

Comment (sputnikci) [inline src/ralph/back_office/admin.py:80]: [Pylint] INFO: Wrong hanging indentation before block (add 4 spaces).
    CustomFieldValueAdminMixin,
    ^   | [bad-continuation]

**Comment by ar4s** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**